.eslintrc

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "eslint:recommended",
   "parserOptions": {
-    "ecmaVersion": 8
+    "ecmaVersion": 2018
```

Comment:
eslint was complaining about

```js
const {x, ...y} = someObj
```

so I bumped it. Side note, I'd love to move this to use a more standard config. [Standard.js](https://standardjs.com/) is my personal favorite, but the front-end team's, airbnb's, or others are good bets too.